app/components/primer/css/variables.pcss

```diff
@@ -0,0 +1,61 @@
+:root {
```

Comment:
Hey @mxriverlynn! I'm not sure we want to move SCSS variables to PVC. I think they will mostly be reserved for utility CSS at this point, and the CSS that we move over from PCSS should use the new set of design tokens that are already being used in `tab_nav`. If there's an issue or discussion you can point me towards that is suggesting we should move all our existing variables to PVC, please let me know!

Reply:
Migrating the SCSS variables to the CSS variables from Primitives is definitely the hardest part. 😆 

There isn't always a clear 1:1 replacement and it depends on the use case. Especially for `margin/padding`. I started a [.scss to .pcss migration guide](https://gist.github.com/simurai/2287ce592d3b1bcb2f30be8b73fac877). It's not really complete and I've just been adding to it while going along. Also might not be 100% accurate. Let me know in case something is unclear.